pkg/server/container_start.go

```diff
@@ -103,7 +103,7 @@ func (c *criContainerdService) startContainer(ctx context.Context, id string, me
 	sandboxConfig := sandboxMeta.Config
 	sandboxID := meta.SandboxID
 	// Make sure sandbox is running.
-	sandboxInfo, err := c.containerService.Info(ctx, &execution.InfoRequest{ContainerID: sandboxID})
+	sandboxInfo, err := c.taskService.Info(ctx, &execution.InfoRequest{ContainerID: sandboxID})
```

Comment:
Because containerd also has `container` notion, which is the metadata + task of the container, it may cause some naming conflict or confusion if we don't follow the same pattern. :)

That's fine, we'll have a separate PR to do the cleanup.